pkg/client/restclient/config.go

```diff
@@ -93,10 +98,10 @@ type Config struct {
 	// on top of the returned RoundTripper.
 	WrapTransport func(rt http.RoundTripper) http.RoundTripper
 
-	// QPS indicates the maximum QPS to the master from this client.  If zero, QPS is unlimited.
+	// QPS indicates the maximum QPS to the master from this client. If zero, it will be set to DefaultQPS: 5.
```

Comment:
nit: "it will be set to DefaultQPS: 5" -> "the created RESTClient will use DefaultQPS: 5". The config doesn't get changed.